ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-main__ocp417.yaml

```diff
@@ -123,7 +123,49 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: multiarch-tuning-operator
     - ref: openshift-e2e-test
-    workflow: openshift-e2e-aws-ovn
+    workflow: ipi-aws-ovn
+- always_run: false
```

Comment:
```suggestion
- optional: true
  run_if_changed: "\.perfscale$"
```

Reply:
@aleskandro This line failed the `make jobs`
```
time="2024-10-12T01:48:10Z" level=fatal msg="failed to parse arguments" error="failed to create CI Op config agent: failed to load config: loading config failed: failed to load ci-operator config (error converting YAML to JSON: yaml: line 128: found unknown escape character)"
```

Reply:
I used instead `run_if_changed: \.perfscale$` and the make jobs passed.
Or should I use `run_if_changed: "\\.perfscale$"`?

Reply:
Hi @qiliRedHat im not sure. Probably what you did is right... but we can change this later if it's not for sure